datastream-kafka-connector/src/main/java/com/linkedin/datastream/connectors/kafka/mirrormaker/KafkaMirrorMakerConnectorTask.java

```diff
@@ -175,8 +191,19 @@ public KafkaMirrorMakerConnectorTask(KafkaBasedConnectorConfig config, Datastrea
 
   @Override
   protected void consumerSubscribe() {
-    LOG.info("About to subscribe to source: {}", _mirrorMakerSource.getTopicName());
-    _consumer.subscribe(Pattern.compile(_mirrorMakerSource.getTopicName()), this);
+    if (_enablePartitionAssignment) {
+      Set<TopicPartition> topicPartition  = getAssignedTopicPartitionFromTask();
+      updateConsumerAssignment(topicPartition);
```

Comment:
I think it might be good idea to log the _consumerAssignment here.

Reply:
There are exisitng logging inside updateConsumerAssignment function.